### ISSUE TITLE
Set .Context.Operation.Id, not only parent-child relationships

### ISSUE THREAD

**Issue by SergeyKanzhelev** [Azure-Samples/e2e-diagnostic-eventhub-ai-function#4]
I'd suggest to set `.Context.Operation.Id` across entire distributed trace so missing telemetry will not break a chain of execution and end-to-end view will be rendered successfully.

**Comment by SLdragon** [comment]
Hi, @SergeyKanzhelev 
Do you mean we need to set Operation Id to the correlationId in the logs?

**Comment by SergeyKanzhelev** [comment]
Yes. This will enable transaction view in Application Insights UI

**Comment by SLdragon** [comment]
Fixed: https://github.com/Azure-Samples/e2e-diagnostic-eventhub-ai-function/pull/8

**Comment by SLdragon** [comment]
Hi, @SergeyKanzhelev , maybe i misunderstood your suggestion, do you mean I also need to add this:
             dependencyTelemetry.Context.Operation.Id = correlationId;
We have already set dependencyTelemetry.Id = correlationId , why we also need to set this?

**Comment by SergeyKanzhelev** [comment]
`dependencyTelemetry.Context.Operation.Id = correlationId` refer to the ID of the overall transaction. `dependencyTelemetry.Id = correlationId` refer to the ID of this specific call. All events of the distributed trace should have the same value in `.Context.Operation.Id` so they can be correlated.

**Comment by SLdragon** [comment]
Thanks @SergeyKanzhelev, I created another pull request to fix this issue, you can have a look.
